spinoso-time/Cargo.toml

```diff
@@ -13,9 +13,18 @@ license = "MIT"
 keywords = ["artichoke", "date", "datetime", "spinoso", "time"]
 categories = ["date-and-time"]
 
+[features]
+default = ["chrono", "tzrs"]
+chrono = ["dep:chrono", "dep:chrono-tz"]
+tzrs = ["dep:once_cell", "dep:regex", "dep:tz-rs", "dep:tzdb"]
+
 [dependencies]
-chrono = { version = "0.4.19", default-features = false, features = ["clock"] }
-chrono-tz = { version = "0.6.0", default-features = false }
+chrono = { version = "0.4.19", default-features = false, features = ["clock"], optional = true }
+chrono-tz = { version = "0.6.0", default-features = false, optional = true }
+once_cell = { version = "1.12.0", optional = true }
+regex =  { version = "1.5.5", default-features = false, features = ["std", "unicode-perl"], optional = true }
+tz-rs = { version = "0.6.9", default-features = false, optional = true }
+tzdb = { version = "0.2.2", default-features = false, features = ["local"], optional = true }
```

Comment:
will bumping this to 0.2.4 fix the minimum versions CI check?

To check this yourself you can do:

```console
$ cargo +nightly generate-lockfile -Zminimal-versions
$ cargo check
```

Reply:
Well, apparently we do actually need `const_fn` after all: https://docs.rs/tz-rs/0.6.9/src/tz/timezone/mod.rs.html#256-269

^-- this is used to generate the fixed timezones. After quickly reading up on `const_fn` (I was ignoring this dep until now), I see it's to help support older versions of rust, so I'm not sure if there is any getting around this apart from forking tz-rs and forcing rust 1.57+ support

Reply:
Okay, found something. `tz-rs` needs `const_fn` >= 0.4.4.

e.g. this fails in tz-rs, but doesn't if `const_fn` is >= 0.4.4.
```
cargo update -Z minimal-versions
cargo check
```

I'll make a PR there (this evening)!

Reply:
awesome! thanks for digging.

I made the PR to `tz-rs` that used `const_fn` in this way to make it optional. Whoops.

Reply:
It looks like your PR was merged upstream and released as 0.6.10. I'll follow up in a new PR and remove the load bearing dep on `const_fn`.